README.md

```diff
@@ -116,6 +112,15 @@ Additional resources:
 * [serverfull](https://github.com/asecurityteam/serverfull)
 * [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
 
+### Gateway Environment Variables
+Here are some environment variables that need to be set, we recommend using the transportd library to handle authentication on outbound requests to Nexpose
```

Comment:
This still reads in a confusing way to me. The variables are still only meaningful when using the sample docker-compose and the sample OpenAPI specification.`Here are some environment variables that need to be set` still suggests that this is part of running the system in production.

I'd suggest starting with more context around the problem (e.g. Nexpose usually requires authentication). Then I'd recommend saying that the sample compose uses transportd as a reverse proxy and that it has a basic auth feature. That could segway into mentioning that our sample OpenAPI specification looks for these variable names when running.

Reply:
can you have another look at the readme? I revised it up a little bit, let me know if it still doesn't make sense, or if there's holes in the explanations